services/cron/tasks_basic.go

```diff
@@ -139,6 +140,20 @@ func registerCleanupHookTaskTable() {
 	})
 }
 
+func registerCleanupPackages() {
+	RegisterTaskFatal("cleanup_packages", &OlderThanConfig{
+		BaseConfig: BaseConfig{
+			Enabled:    true,
+			RunAtStart: true,
```

Comment:
Are you talking about #19221? If yes, that's already changed.

Reply:
Ah, that value is simply the default value if the setting is missing, correct?
In that case ignore my comment, I thought it would mean that it would always run at start.